test/app.test.js

```diff
@@ -30,7 +30,7 @@ describe('app', function() {
       return redirects.indexOf(route) === -1;
     }).map(function(url) {
       if (url === '/') return '/';
-      return '/' + url;
+      return '/' + url + '/';
```

Comment:
so it looks like this is being caused by the `app.use('/', express.static` call being "too early" in the chain. Static doesn't have a clean `next()` fallthrough so it does unexpected things. Can we try to keep the static bind just before the 404?